src/DssSpell.sol

```diff
@@ -16,197 +15,116 @@
-    // GovAlpha Core Unit
-    address constant GOV_WALLET     = 0x01D26f8c5cC009868A4BF66E268c17B057fF7A73;
-    // Growth Core Unit
-    address constant GRO_WALLET     = 0x7800C137A645c07132886539217ce192b9F0528e;
-    // Marketing Content Production Core Unit
-    address constant MKT_WALLET     = 0xDCAF2C84e1154c8DdD3203880e5db965bfF09B60;
-    // Oracles Core Unit
-    address constant ORA_WALLET     = 0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf;
-    // Protocol Engineering
-    address constant PE_WALLET      = 0xe2c16c308b843eD02B09156388Cb240cEd58C01c;
-    // Risk Core Unit
-    address constant RISK_WALLET    = 0xd98ef20520048a35EdA9A202137847A62120d2d9;
-    // Real-World Finance Core Unit
-    address constant RWF_WALLET     = 0x9e1585d9CA64243CE43D42f7dD7333190F66Ca09;
-    // Ses Core Unit
-    address constant SES_WALLET     = 0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6;
-
-    uint256 constant MAY_01_2021 = 1619827200;
-    uint256 constant JUN_21_2021 = 1624233600;
-    uint256 constant JUL_01_2021 = 1625097600;
-    uint256 constant SEP_01_2021 = 1630454400;
-    uint256 constant SEP_13_2021 = 1631491200;
-    uint256 constant SEP_20_2021 = 1632096000;
-    uint256 constant OCT_01_2021 = 1633046400;
-    uint256 constant NOV_01_2021 = 1635724800;
-    uint256 constant JAN_01_2022 = 1640995200;
-    uint256 constant MAY_01_2022 = 1651363200;
-    uint256 constant JUL_01_2022 = 1656633600;
-    uint256 constant SEP_01_2022 = 1661990400;
-
-    uint256 constant MILLION = 10 ** 6;
-    uint256 constant WAD     = 10 ** 18;
-
-    // Turn off office hours
+        "2021-09-17 MakerDAO Executive Spell | Hash: 0x613a50aee82adca6f6f32a4f7298c70b1c9d061648e786c84b3872ffc5963e3e";
+
+    // Many of the settings that change weekly rely on the rate accumulator
+    // described at https://docs.makerdao.com/smart-contract-modules/rates-module
+    // To check this yourself, use the following rate calculation (example 8%):
+    //
+    // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
+    //
+    // A table of rates can be found at
+    //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
+    //
+    uint256 constant ONE_PCT_RATE = 1000000000315522921573372069;
+
+    // Math
+    uint256 constant THOUSAND = 10 ** 3;
+    uint256 constant MILLION  = 10 ** 6;
+    uint256 constant BILLION  = 10 ** 9;
+    uint256 constant RAY      = 10 ** 27;
+
+    address constant GUNI                   = 0xAbDDAfB225e10B90D798bB8A886238Fb835e2053;
+    address constant MCD_JOIN_GUNI_A        = 0xbFD445A97e7459b0eBb34cfbd3245750Dba4d7a4;
+    address constant MCD_CLIP_GUNI_A        = 0x5048c5Cd3102026472f8914557A1FD35c8Dc6c9e;
+    address constant MCD_CLIP_CALC_GUNI_A   = 0x25B17065b94e3fDcD97d94A2DA29E7F77105aDd7;
+    address constant PIP_GUNI               = address(0); // TODO
+
+    // Turn on office hours
     function officeHours() public override returns (bool) {
```

Comment:
I think we don't need to override the `officeHours` modifier for office hours on, as they're on by [default](https://github.com/makerdao/dss-exec-lib/blob/master/src/DssAction.sol#L39).

Reply:
Right, it would be enough removing the method, but if wants to be left for explicitness not bad either.

Reply:
I didn’t know, but I like the explicitness